Code-review/index.html

```diff
@@ -0,0 +1,197 @@
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    <br>
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
+            </section>
+            <img  class="myimg" src="https://th.bing.com/th/id/OIP.Jciq-y05-b4-VwDB_iUpQwHaEH?w=300&h=180&c=7&r=0&o=5&dpr=1.3&pid=1.7" alt="image1"/>
+            <section class="section1">
+                <div class="info">
+                    <span style="font-weight: bold;">
+                        Contributing to open source part II
+                    </span>
+                    <br>
+                    
+                    <span style="margin-top:5px;">The odin Project</span>
+                    <br>
+                    <span>38k views .<span> 2 years ago</span></span>
+                    
+                </div>
```

Comment:
Please i would kindly suggest that you try to avoid using inline styling as this makes the code look unclean. it would be better if you give them classes and do the styling from the CSS file.😄
Please do this for all the cases in the file.